src/server/cluster/cluster_family.cc

```diff
@@ -735,11 +748,9 @@ void ClusterFamily::InitMigration(CmdArgList args, ConnectionContext* cntx) {
   if (auto err = parser.Error(); err)
     return cntx->SendError(err->MakeReply());
 
-  auto* node =
-      CreateIncomingMigration(cntx->conn()->RemoteEndpointAddress(), port, std::move(slots));
-
   VLOG(1) << "Init migration " << cntx->conn()->RemoteEndpointAddress() << ":" << port;
-  node->Init(sync_id, flows_num);
+
+  CreateIncomingMigration(cntx->conn()->RemoteEndpointAddress(), port, std::move(slots), flows_num);
```

Comment:
Actually, if all nodes store the full config, we can get the remote endpoint address by the node id from the config 🤔 We don't have this problem like with replicaof where an external address of th master relative to replica is explicitly specified

Reply:
I will remove port and endpoint address in one of the next PR. I'm going to use endpoint_ID instead